### PR TITLE
Doc: Block API > Registration: switch markdown to a-tags

### LINKED ISSUE #50109: Documentation: Use HTML for links on call out
### Description

The page Registration has a callout section on the top. 
The links are in Markdown, and need to be probably a HTML `<a href` tag. 
Screenshot 2023-04-26 at 18 43 57

Screenshot 2023-04-26 at 18 41 02

Occured in #49792

See [Registration documentation page](https://developer.wordpress.org/block-editor/reference-guides/block-api/block-registration/)

### PR DESCRIPTION
**Pull request by bph**
## What?
fixes #50109

## Why?
in `<div callouts>` Links need to be a-tags instead of Markdown 
That is only a theory, I am testing. I have not found any information in that regard in the documentation. 

## How?
switch markdown to a-tags

### CODE REVIEW

**Comment by bph** [comment]
apologies for the misleading branch name. should be docs/update-block-registration

File: docs/reference-guides/block-api/block-registration.md [hunk 1]
```diff
@@ -3,9 +3,8 @@
 Block registration API reference.
 
 <div class="callout callout-alert">
-You can use the functions documented on this page to register a block with JavaScript only on the client, but the recommended method is to register new block types also with PHP on the server using the `block.json` metadata file. See [metadata documentation for complete information](/docs/reference-guides/block-api/block-metadata.md).
-
-[Learn how to create your first block](/docs/getting-started/create-block/README.md) for the WordPress block editor. From setting up your development environment, tools, and getting comfortable with the new development model, this tutorial covers all you need to know to get started with creating blocks.
+You can use the functions documented on this page to register a block with JavaScript only on the client, but the recommended method is to register new block types also with PHP on the server using the `block.json` metadata file. See <a href="/docs/reference-guides/block-api/block-metadata.md)">metadata documentation for complete information</a>
```
Comment (carolinan) [inline docs/reference-guides/block-api/block-registration.md]:
```suggestion
You can use the functions documented on this page to register a block with JavaScript only on the client, but the recommended method is to register new block types also with PHP on the server using the `block.json` metadata file. See <a href="/docs/reference-guides/block-api/block-metadata.md">metadata documentation for complete information</a>.
```

File: docs/reference-guides/block-api/block-registration.md [hunk 2]
```diff
@@ -3,9 +3,8 @@
...
+<a href="/docs/getting-started/create-block/README.md">Learn how to create your first block</a>() for the WordPress block editor. From setting up your development environment, tools, and getting comfortable with the new development model, this tutorial covers all you need to know to get started with creating blocks.
```
Comment (carolinan) [inline docs/reference-guides/block-api/block-registration.md]:
```suggestion
<a href="/docs/getting-started/create-block/README.md">Learn how to create your first block</a> for the WordPress block editor. From setting up your development environment, tools, and getting comfortable with the new development model, this tutorial covers all you need to know to get started with creating blocks.
```

Comment (bph) [reply docs/reference-guides/block-api/block-registration.md]: Thanks for catching things, @carolinan

File: docs/reference-guides/block-api/block-registration.md (hunk 1 above)
Comment (bph) [reply docs/reference-guides/block-api/block-registration.md]: Updated with your suggestions

**Review by gziolo — APPROVED** [review]
Let's give it a try. I wasn't aware that everything inside a `div` needs to be HTML 😞

**Comment by bph** [comment]
Thanks @gziolo, as mentioned, it's just a theory, I am testing :-) 

It also seems there is one test failing, but also that it might not be connected to my PR. When you get a minute, could you help me figure out what the next steps are to get this merged?

**Comment by gziolo** [comment]
The failing test was unrelated, so I pushed the changes using my extended permissions 😅

**Comment by bph** [comment]
Thank you! 🤗

**Comment by bph** [comment]
It worked: https://developer.wordpress.org/block-editor/reference-guides/block-api/block-registration/
Screenshot 2023-04-27 at 12 33 35

**Comment by gziolo** [comment]
I see now that links are broken, so we probably need also to use full URLs here or tweak the processing for the handbook page that happens for the developer portal.

**Comment by bph** [comment]
I also found the links nor working. I  try to figure out what the right link would be [in this PR](https://github.com/WordPress/gutenberg/pull/50131)

**Pull request merged**